examples/example_cori_haswell.cfg

```diff
@@ -0,0 +1,128 @@
+
+  [[ atm_monthly_180x360_aave ]]
+  input_files = "eam.h0"
+
+  [[ atm_daily_180x360_aave ]]
+  frequency = "daily"
+  input_files = "eam.h1"
+  vars = "PRECT"
+
+  [[ atm_monthly_glb ]]
+  input_files = "eam.h0"
+  input_subdir = "archive/atm/hist"
+  mapping_file = "glb"
+  years = "51:61:5",
+
+  [[ land_monthly ]]
+  input_files = "elm.h0"
+  input_subdir = "archive/lnd/hist"
+  vars = "FSH,LAISHA,LAISUN,RH2M"
+
+  [[ rof_monthly ]]
+  extra_vars = 'areatotal2'
+  input_files = "mosart.h0"
+  input_subdir = "archive/rof/hist"
+  mapping_file = ""
+  vars = "RIVER_DISCHARGE_OVER_LAND_LIQ"
+
+[tc_analysis]
+active = True
+scratch = /global/cscratch1/sd/forsyth
+years = "51:53:2",
+
+[e3sm_diags]
+active = True
+grid = '180x360_aave'
+obs_ts = /global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/time-series
+ref_final_yr = 2014
+ref_start_yr = 1985
+reference_data_path = /global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/climatology
+short_name = '20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis'
+ts_num_years = 2
+years = "51:55:2", "51:55:4",
+
+  [[ atm_monthly_180x360_aave ]]
+  climo_diurnal_frequency = "diurnal_8xdaily"
+  climo_diurnal_subsection = "atm_monthly_diurnal_8xdaily_180x360_aave"
+  dc_obs_climo = "/compyfs/e3sm_diags_data/obs_for_e3sm_diags/climatology"
+  sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow", "zonal_mean_2d_stratosphere",
+  streamflow_obs_ts = /global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/time-series
+
+  [[ atm_monthly_180x360_aave_tc_analysis ]]
+  # Running as its own subtask because tc_analysis requires jobs to run sequentially, which slows down testing
+  sets = "tc_analysis",
+  tc_obs = /global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/tc_analysis
+  years = "51:53:2",
+
+  [[ atm_monthly_180x360_aave_mvm ]]
+  # Test model-vs-model using the same files as the reference
+  climo_diurnal_frequency = "diurnal_8xdaily"
+  climo_diurnal_subsection = "atm_monthly_diurnal_8xdaily_180x360_aave"
+  climo_subsection = "atm_monthly_180x360_aave"
+  diff_title = "Difference"
+  gauges_path = /global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/time-series/GSIM/GSIM_catchment_characteristics_all_1km2.csv
+  ref_final_yr = 52
+  ref_name = "20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis"
+  ref_start_yr = 51
+  ref_years = "51-52",
+  reference_data_path = /global/cscratch1/sd/forsyth/zppy_complete_run_nersc_output/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis/post/atm/180x360_aave/clim
+  reference_data_path_climo_diurnal = /global/cscratch1/sd/forsyth/zppy_complete_run_nersc_output/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis/post/atm/180x360_aave/clim_diurnal_8xdaily
+  reference_data_path_tc = /global/cscratch1/sd/forsyth/zppy_complete_run_nersc_output/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis/post/atm/tc-analysis_0051_0052
```

Comment:
Presumably, this change from `51_52` to `0051_0052` will get TC working on `e3sm_diags_atm_monthly_180x360_aave_mvm_model_vs_model_0051-0052_vs_0051-0052`.